unfold_studio/static/app.js

```diff
@@ -91,6 +91,39 @@ define(
                     return false;
                 });
             });
+
+            // Mutation observer to update async generate calls
```

Comment:
Nice! So basically every time there's a change to `#player` (new content hits the screen), we'll check to see whether there's a generate target; if there is, fill it with the cached content.